App/static/css/estilos.css

```
@@ -0,0 +1,58 @@
+body {
+  background-color: rgba(185, 196, 185, 0.2);
+}
+
+.calc-container {
+  display: block;
+}
+
+h2 {
+  text-align: center;
+}
+
+.calc {
+  margin: 0 auto;
+  padding: 20px;
+  background-color: rgba(63, 80, 241, 0.4);
+  width: 300px;
+  height: 300px;
+  border-radius: 10px;
+  padding: 20px;
+  user-select: none;
+}
+
+.calc, td {
+  border-collapse: collapse;
+  padding: 5px;
+}
+
+.calc td button {
+  display: block;
+  background-color: white;
+  width: 60px;
+  height: 60px;
+  font-size: 20px;
+}
+
+#resultado {
```

Is a good practice to use a class selector, not Id one. 
```suggestion
#resultado {
.resultado {
```

---

done

---

done